src/services/admin/email.service.ts

```diff
@@ -0,0 +1,67 @@
+import { dataSource } from '../../configs/dbConfig'
+import Email from '../../entities/email.entity'
+import { EmailStatusTypes } from '../../enums'
+
+export const createEmailTemplate = async (
+  subject: string,
+  content: string
+): Promise<{
+  statusCode: number
+  emailTemplate?: Email | null
+  message: string
+}> => {
+  try {
+    const emailRepositroy = dataSource.getRepository(Email)
+
+    const newEmailTemplate = new Email(
+      '',
+      subject,
+      content,
+      EmailStatusTypes.SENT
+    )
+
+    const savedEmailTemplate = await emailRepositroy.save(newEmailTemplate)
```

Comment:
Can you check the ERD here? https://drive.google.com/file/d/11KMgdNu2mSAm0Ner8UsSPQpZJS8QNqYc/view

Email templates are saved under the platform table. The email table is used to track the email that we are sending through the platform.

Reply:
Any updates @sathudeva7?

Reply:
I have an issue on how to find platform to add the email templates in the platform

Reply:
we have created a Platform entity. So you can use it the the same as you have used the emailRepositroy @sathudeva7

Reply:
yes but how can we find the particular platform from db, because we didnt pass any platformId in the request query

Reply:
we had a discussion regarding this. Can you create a new table to store the email templates? I will update the ERD as well.

Reply:
sure